lib/dotcom_web/plugs/path_params_redirector.ex

```diff
@@ -0,0 +1,39 @@
+defmodule DotcomWeb.Plugs.PathParamsRedirector do
+  import Plug.Conn, only: [put_status: 2, halt: 1]
+  import Phoenix.Controller, only: [redirect: 2]
+
+  @behaviour Plug
+
+  @impl true
+
+  @spec init(Keyword.t()) :: Keyword.t()
+  def init(opts) do
+    opts
+    |> validate_to_keyword()
```

Comment:
Could just move this to the init.

```
def init([to: _ = opts), do: opts

def init(_), do: raise(...)
```

Reply:
[Fair](https://github.com/mbta/dotcom/pull/2432/commits/9b6d5ab6571ffd0a0e24a06f1cb7e9d41d01c397)